specs.md

```
@@ -0,0 +1,94 @@
+| `actions/download-artifact` | `d3f86a106a0bac45b974a628896c90dbdf5c8093` |
+| `github/codeql-action/upload-sarif` | `865f5f5c36632f18690a3d569fa0a764f2da0c3e` |
+| `softprops/action-gh-release` | `3bb12739c298aeb8a4eeaf626c5b8d85266b0e65` |
+| `actions/github-script` | `f28e40c7f34bde8b3046d885e986cb6290c5673b` |
+
+## Acceptance Criteria
+
+### AC1: All `uses:` declarations use SHA references
+Every GitHub Action `uses:` declaration in the four workflow files must end with a 40-character hexadecimal SHA (not a version tag like `@v4` or `@v2`).
+
+**Verification:**
+```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)